src/pages/DownloadMonitor.vue

```diff
@@ -83,6 +83,14 @@
 
                             </div>
                         </div>
+                        <button
+                            v-if="downloadObject.failed"
+                            class="button download-item-action-button"
+                            v-tooltip.left="'Retry'"
+                            @click="$store.dispatch('download/retryDownload', downloadObject)"
+                        >
+                            <i class="fas fa-redo" />
```

Comment:
IMO the retry icon is now comically large when compared to rest of the UI (even if it's the same size in pixels as the remove icon, it's visually larger).